api/AudioScheduledSourceNode.json

```diff
@@ -10,8 +10,9 @@
             },
             {
               "version_added": "14",
-              "version_removed": "58",
-              "alternative_name": "AudioSourceNode"
```

Comment:
I don't think there's any point to us documenting that there was a `AudioSourceNode` parent interface, since it didn't have any members. Since it even existing at the same time as `AudioScheduledSourceNode` briefly, it wasn't even a matter of a straight rename.